SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -662,7 +662,7 @@ void ASpatialFunctionalTest::CrossServerNotifyStepFinished_Implementation(ASpati
 
 	const FString FlowControllerDisplayName = FlowController->GetDisplayName();
 
-	UE_LOG(LogSpatialGDKFunctionalTests, Display, TEXT("%s finished Step"), *FlowControllerDisplayName);
+	UE_LOG(LogSpatialGDKFunctionalTests, Display, TEXT("%s finished Step in %fs"), *FlowControllerDisplayName, TimeRunningStep);
```

Comment:
Does this work? The last time I looked at this, it was always 0 for most servers/clients, I think.

Reply:
I think it depends how much work is done on a teststep, because it's divided into ticks, so if something takes less than a tick (which I think it 1/60 of a second) it will show up as 0